packages/commonwealth/client/scripts/controllers/app/login.ts

```diff
@@ -187,6 +183,8 @@ export async function updateActiveAddresses({
     shouldRedraw
   );
 
+  getTokenBalance();
```

Comment:
Are there any other times we need to query token balance, such as re-visiting the site when already logged in, or switching to a new community (given that getTokenBalance() only works on the active community)?

Reply:
This is the main place where tokenbalance is always loaded. It will still be loaded when a user is logged in b/c it is within 'setActiveAddresses'(its not called on login specifically) which is part of chain initialization which should always happen when visiting the site, switching communities, etc. 

I did however do a quick dive into every where tokenBalance was called prior. An only `LoginSelector` came up so I also added the call there as well now to cover old cases(only fires if it hasnt fired already). Other than that there is no other usage of tokenBalance calls